subber/util.py

```diff
@@ -23,17 +23,13 @@ def utc_epoch_sec_to_years(sec):
     logging.debug('Converting {} seconds from UTC epoch to years to '
                   'date'.format(sec))
 
-    try:
-        # Elapsed seconds
-        elapsed_sec = datetime.now() - sec
-
-        # Elapsed days from time
-        days = datetime.timedelta(seconds=elapsed_sec).days
-
-        # Reddit rounds years this way on their website
-        # Subber does the same for consistency
-        return int(days/365)
-
-    except Exception:
+    if not isinstance(sec, (int, float)):
         logging.error('Unable to calculate years to date from UTC epoch '
                       'timestamp')
+        return -1
+
+    time = datetime.datetime.utcnow() - datetime.datetime.utcfromtimestamp(sec)
```

Comment:
Minor nit: why not `from datetime import datetime` and do `datetime.utcnow() - datetime.utcfromtimestamp(sec)`